litmus-portal/frontend/src/redux/reducers/workflow.ts

```diff
@@ -59,4 +65,14 @@ export const workflowData = createReducer<WorkflowData>(initialState, {
   },
 });
 
-export default workflowData;
+export const workflowManifest = createReducer<WorkflowManifest>(init, {
+  [WorkflowActions.SET_WORKFLOW_MANIFEST](
+    state: WorkflowManifest,
+    action: WorkflowAction
+  ) {
+    return {
+      ...state,
+      ...(action.payload as Object),
```

Comment:
as Object is not required....as the payload is of an appropriate type of WorkflowManifest

Reply:
This was intentional since there is a possibility of sending both engine YAML and manifest (updated with enigneYAML) as payload.

Reply:
We have the data as WorkflowManifest type, this spreading action.payload would work as well, you're right, making the change, thanks for the suggestion.